zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -2601,6 +2604,7 @@ search.toolbar.label.type.header = Header
 search.toolbar.label.type.request = Request
 search.toolbar.label.type.response = Response
 search.toolbar.label.type.url = URL
+search.toolbar.prompt.regex = Search term or regular expression
```

Comment:
Not sure "term or" is clear, does it mean it accepts plain text? Fine as is if others are happy with it though.

Reply:
Ya that was supposed to be the implication.